api/tests/Feature/ApplicantTest.php

```diff
@@ -52,26 +52,53 @@ public function testCountApplicantsQuery(): void
     {
         // Get the ID of the base admin user
         $user = User::All()->first();
-        $pool1 = Pool::factory()->candidatesAvailableInSearch()->create([
+        $ITPool1 = Pool::factory()->candidatesAvailableInSearch()->create([
+            'user_id' => $user['id'],
```

Comment:
I think you also need to set the classification of these pools

Reply:
I could set a classification but the scope just checks the pools publishing group.

Reply:
I see 👍